Content.Client/GameObjects/Components/Research/ResearchClientServerSelectionMenu.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using Content.Shared.GameObjects.Components.Research;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Shared.IoC;
+using Robust.Shared.Maths;
+
+namespace Content.Client.GameObjects.Components.Research
+{
+    public class ResearchClientServerSelectionMenu : SS14Window
+    {
+        private ItemList _servers;
+
+        protected override Vector2? CustomSize => (300, 300);
+        public ResearchClientBoundUserInterface Owner { get; set; }
+
+        public ResearchClientServerSelectionMenu()
+        {
+            IoCManager.InjectDependencies(this);
```

Comment:
you don't seem to be using any IoC'd variable in here, this is doing nothing

Reply:
Whoops. I put those there in case I needed to inject dependencies, but forgot to remove them later. Fixed now.